arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -1,20 +1,26 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Net;
 using System.Text;
 using System.Threading.Tasks;
 using ArangoDBNetStandard;
 using ArangoDBNetStandard.CollectionApi;
+using ArangoDBNetStandard.DocumentApi;
 using Xunit;
 
 namespace ArangoDBNetStandardTest.CollectionApi
 {
     public class CollectionApiClientTest : IClassFixture<CollectionApiClientTestFixture>
     {
         private CollectionApiClient _collectionApi;
+        private readonly CollectionApiClientTestFixture _fixture;
+        private readonly DocumentApiClient _documentApi;
 
         public CollectionApiClientTest(CollectionApiClientTestFixture fixture)
         {
+            _fixture = fixture;
```

Comment:
From what I saw, `_fixture` is used only to get the `TestCollection` property. Maybe replace `_fixture` with `_testCollection`? That's what Ross did in #25 and it gives a clearer intent, in my opinion.

Reply:
Actually forget that, it's perfectly fine to include the whole fixture class as it's the data for the class test.